samples/nrf_rpc/protocols_serialization/client/src/rpc_utils_shell.c

```diff
@@ -134,6 +160,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 #if defined(CONFIG_NRF_RPC_UTILS_SYSTEM_HEALTH)
 	SHELL_CMD_ARG(system_health, NULL, "Get system health", cmd_system_health, 0, 0),
 #endif
+	SHELL_CMD_ARG(rpc, NULL, "Control RPC subsystem ", cmd_rpc, 2, 0),
```

Comment:
Using `rpc` as subcommand would result in usage like: `rpc rpc 1`, maybe we can use something like `control`? e.g. `rpc control enable/disable` (these are parsed properly by `shell_strtobool()`)